cosmos/templates/photo_album/photo_album_view.html

```diff
@@ -0,0 +1,80 @@
+{% extends "subpage.html" %}
+
+{% load crispy_forms_tags %}
+
+{% block title %}{{ album.title }} - {{ album.date }}{% endblock title %}
+
+{% block content %}
+
+<div class="container">
+    <div class="w-75 mx-auto">
+        <div class="d-flex justify-content-between">
+            <h2>{{ album.title }} - {{ album.date }}</h2>
+            {% if perms.cosmos.add_photoobject and perms.cosmos.delete_photoalbum %}
+            <div>
+                <a class="btn btn-cosmos-primary" href="{% url 'photo_album-add_photos' album.id %}">
+                    <i class="bi bi-plus-lg"></i>
+                    Add new Photos
+                </a>
+                <a class="btn fs-4 text-danger" href="{% url 'photo_album-delete' album.id %}">
+                    <i class="bi bi-trash"></i>
+                </a>
+            </div>
+            {% endif %}
```

Comment:
will we have a case where a user has adding permissions but not deleting permissions or vice versa? if so, it's best to have the inclusion checks surrounding the individual buttons.

Reply:
Should never happen, as the user should have both permissions at the same time, as handling the permissions on such specific level is too much effort, and gives no benefit. Users will either have all permissions on an object, or none.